edge/pkg/edged/edged.go

```diff
@@ -697,6 +721,23 @@ func (e *edged) syncLoopIteration(plegCh <-chan *pleg.PodLifecycleEvent, houseke
 	}
 }
 
+// NewNamespacedNameFromString parses the provided string and returns a NamespacedName.
+// The expected format is as per String() above.
```

Comment:
where is the String() method above ? which is the expected format ?

Reply:
Thanks, will remove